src/mixins/name-request-mixin.ts

```diff
@@ -63,7 +63,8 @@ export default class NameRequestMixin extends Mixins(DateMixin) {
       !!this.getNrApprovedName(nr) &&
       nr.nrNum &&
       nr.requestTypeCd &&
-      [NameRequestTypes.CHANGE_OF_NAME, NameRequestTypes.CONVERSION].includes(nr.request_action_cd)
+      [NameRequestTypes.CHANGE_OF_NAME, NameRequestTypes.CONVERSION, NameRequestTypes.NEW]
+        .includes(nr.request_action_cd)
```

Comment:
Could this code change accidentally allow a "new" NR for a BEN IA correction, which previously didn't allow this type?

Reply:
Latest code has special case for firms so it no longer impacts others.